Version 3.23.4 - Cannot find module  [...] zod-to-json-schema/dist/esm/parsers/branded

Hey,

The latest version does not work (3.23.4)

https://stackblitz.com/edit/stackblitz-starters-jhxkjr?file=package.json

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/projects/stackblitz-starters-jhxkjr/node_modules/zod-to-json-schema/dist/esm/parsers/branded' imported from /home/projects/stackblitz-starters-jhxkjr/node_modules/zod-to-json-schema/dist/esm/parsers/record.js
```

me too:

```
Cannot find module '/Users/robson.junior/dev/site/node_modules/zod-to-json-schema/dist/esm/parsers/branded' imported from /Users/robson.junior/dev/site/node_modules/zod-to-json-schema/dist/esm/parsers/record.js
  Stack trace:
    at finalizeResolution (node:internal/modules/esm/resolve:265:11)
    at defaultResolve (node:internal/modules/esm/resolve:1157:11)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:87:39)
```

--- a/src/parsers/record.ts
+++ b/src/parsers/record.ts
@@ -9,7 +9,7 @@ import { Refs } from "../Refs.js";
 import { JsonSchema7EnumType } from "./enum.js";
 import { JsonSchema7ObjectType } from "./object.js";
 import { JsonSchema7StringType, parseStringDef } from "./string.js";
-import { parseBrandedDef } from "./branded";
+import { parseBrandedDef } from "./branded.js";
 
 type JsonSchema7RecordPropertyNamesType =
   | Omit<JsonSchema7StringType, "type">

